Lmod/branches/7.0/lmod-bash-xtrace.patch

```diff
@@ -0,0 +1,20 @@
+--- Lmod-6.6/init/bash.in	2016-12-15 17:25:30.645403685 +0100
++++ Lmod-6.6-new/init/bash.in	2016-12-15 17:12:17.217391931 +0100
+@@ -1,4 +1,7 @@
+ # -*- shell-script -*-
++xtrace_saved="$(set -o | grep xtrace | grep 'on')";
++set +x
++
+ if [[ ${-/x} != $- ]]; then
+    echo "Start of Lmod init/bash script to define the module command"
+ fi
+@@ -137,6 +140,9 @@
+    echo "End of Lmod init/bash script to define the module command"
+ fi
```

Comment:
i'm with the impression that this `fi` is missing its corresponding `fi`;
in any case, we need to advance with testing this asap, therefor merging!